src/main/resources/file-download-context.xml

```diff
@@ -73,13 +77,9 @@
 		<value>HmdbProteins</value>
 		<value>HmdbMolecules</value>
 		<value>RHEAIdentifierToReactionlikeEvent</value>
-		<value>ComplexPortal</value>
-		<value>ComplexPortalSARS</value>
-		<value>ComplexPortalSARSCOV2</value>
-		<value>IUPHAR</value>
+		<!-- <value>IUPHAR</value> --><!-- Renamed to GuideToPharmacologyLigands -->
```

Comment:
Same as here: https://github.com/reactome/add-links/pull/172/files?diff=unified&w=1#r657275281.  Moving it to just above "GuideToPharmacologyLigands" so the replacement is immediately after it.